### ISSUE TITLE
Running bpsh without the shell on Ubuntu throws Java Preferences error on shutdown

### ISSUE THREAD

**Issue by sage-t** [SpectraLogic/ds3_script#46]
On Ubuntu 16.04, the commands `bpsh -v`, `bpsh -h`, and `bpsh <script>` give:

(Although if debug mode or logging is enabled when running a script, everything goes fine.)

```
Exception in thread "Thread-2" java.lang.ExceptionInInitializerError
	at java.util.prefs.FileSystemPreferencesFactory.userRoot(FileSystemPreferencesFactory.java:41)
	at java.util.prefs.Preferences.userRoot(Preferences.java:454)
	at java_util_prefs_Preferences$userRoot.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
	at com.spectralogic.dsl.helpers.Globals.<clinit>(Globals.groovy:38)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray$1.run(CallSiteArray.java:67)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray$1.run(CallSiteArray.java:64)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.createCallStaticSite(CallSiteArray.java:64)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.createCallSite(CallSiteArray.java:161)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:128)
	at com.spectralogic.dsl.Tool$_main_closure1.doCall(Tool.groovy:28)
	at com.spectralogic.dsl.Tool$_main_closure1.doCall(Tool.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:104)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:326)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:264)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1041)
	at groovy.lang.Closure.call(Closure.java:421)
	at groovy.lang.Closure.call(Closure.java:415)
	at groovy.lang.Closure.run(Closure.java:502)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalStateException: Shutdown in progress
	at java.lang.ApplicationShutdownHooks.add(ApplicationShutdownHooks.java:66)
	at java.lang.Runtime.addShutdownHook(Runtime.java:211)
	at java.util.prefs.FileSystemPreferences$4.run(FileSystemPreferences.java:439)
	at java.util.prefs.FileSystemPreferences$4.run(FileSystemPreferences.java:437)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.util.prefs.FileSystemPreferences.<clinit>(FileSystemPreferences.java:437)
	... 31 more
```

**Comment by sage-t** [comment]
Also happens on Red Hat

**Comment by sage-t** [comment]
Also on Mac

**Comment by gtrossell** [comment]
Also on Ubuntu 18.04

**Comment by sage-t** [comment]
Fixed in v3.5.1
